package.json

```diff
@@ -5,6 +5,12 @@
     "name": "Medhat Dawoud",
     "email": "medhatdawoud@gmail.com"
   },
+  "contributors": [
```

Comment:
don't add this here, I'm using all contributors bot to add contributors, you shouldn't add yourself, I'll add you after merging some contributions from you